### DISCUSSION TITLE
Channels: ReadRange / WriteRange

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#66536] [Q&A] [answered]
Are there any performance gains that might be achieved if channels had a ReadRange / WriteRange?
I'm not sure about the current implementation, but my though is that every time you read or write, you likely acquire some lock-like resource and if there was a batch method, then maybe things would move faster.

**Comment by stephentoub** [comment, marked as answer]
Duplicate of https://github.com/dotnet/runtime/issues/24034
